<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Historial</class>
 <widget class="QWidget" name="Historial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4">
    <widget class="QPushButton" name="cboxAdelanteButton">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSpinBox" name="yearPicker">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="monthPicker">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="maxVisibleItems">
        <number>12</number>
       </property>
       <item>
        <property name="text">
         <string>Enero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Febrero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marzo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Abril</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mayo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Junio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Julio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Agosto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Septiembre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Octubre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Noviembre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diciembre</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="weekPicker">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fortnightPicker">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="rangoInicio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="locale">
        <locale language="Spanish" country="Mexico"/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>d/MMM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="rangoFin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="locale">
        <locale language="Spanish" country="Mexico"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="displayFormat">
        <string>d/MMM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="empleadosCbox"/>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="exportarButton">
     <property name="text">
      <string>Exportar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="cboxPeriodo">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>Año</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Quincena</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Semana</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Día</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rango</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="cboxAtrasButton">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <layout class="QHBoxLayout" name="totalesLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="demoraLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="antelacionLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="faltasLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>yearPicker</tabstop>
  <tabstop>monthPicker</tabstop>
  <tabstop>weekPicker</tabstop>
  <tabstop>rangoInicio</tabstop>
  <tabstop>rangoFin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
